lib/active_merchant/billing/gateways/wompi.rb

```diff
@@ -0,0 +1,249 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class WompiGateway < Gateway
+      self.test_url = 'https://sandbox.wompi.co/v1'
+      self.live_url = 'https://production.wompi.co/v1'
+
+      self.supported_countries = ['CO']
+      self.default_currency = 'COP'
+      self.supported_cardtypes = [:visa, :master, :american_express, :discover]
+
+      self.homepage_url = 'https://wompi.co/'
+      self.display_name = 'Wompi'
+
+      self.money_format = :cents
+
+      PAYMENT_SOURCE_TYPES = %w[ CARD NEQUI ]
+
+      def initialize(options={})
+        requires!(options, :public_key, :private_key)
+
+        super
+      end
+
+      def query_acceptance_token
+        action = "/merchants/#{@options[:public_key]}"
+
+        commit(:get, action)
+      end
+
+      def store(payment_method, options={})
+        post = {}
+
+        post[:number] = payment_method.number
+        post[:cvc] = payment_method.verification_value
+        post[:exp_month] = format(payment_method.month, :two_digits)
+        post[:exp_year] = format(payment_method.year, :two_digits)
+        post[:card_holder] = payment_method.name
+
+        commit(:post, '/tokens/cards', post)
+      end
+
+      def purchase(money, payment, options={})
+        post = {}
+        post[:acceptance_token] = query_acceptance_token.authorization
+
+        add_invoice(post, money, options)
+        add_payment(post, payment, options)
+        add_address(post, payment, options)
+        add_customer_data(post, options)
+
+        commit(:post, '/transactions', post)
+      end
+
+      # Lookup transaction using reference or using transaction id path parans
+      #
+      def query_transaction(reference, options={})
+        if options[:path_params] && reference.present?
+          commit(:get, "/transactions/#{reference}")
+        else
+          commit(:get, "/transactions?reference=#{reference}")
+        end
+      end
+
+      def pse_financial_institutions
+        commit(:get, "/pse/financial_institutions")
+      end
+
+      def supports_scrubbing?
+        true
+      end
+
+      def scrub(transcript)
+        transcript.
+          gsub(/(Authorization: Bearer )([^\s])+/, '\1[FILTERED]').
```

Comment:
`\S` matches non-whitespace:

```suggestion
          gsub(/\b(Authorization:\s+Bearer\s+)\S+/, '\1[FILTERED]').
```